apps/Minecraft Pi (Modded)/install-64

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+version="191/artifact/out/minecraft-pi-reborn-client-2.3.3"
+
+# Remove old repos
+sudo rm -f /etc/apt/{sources.list,trusted.gpg}.d/mcpi-revival.{list,gpg} &>/dev/null || true
+sudo rm -f /etc/apt/{sources.list,trusted.gpg}.d/mcpiscript.{list,gpg} &>/dev/null || true
+
+
+# Make planet directory, to put the AppImage into
+mkdir ~/.planet-launcher
+
+echo "Downloading the AppImage"
+wget https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/${version}-arm64.AppImage -O ~/.planet-launcher/minecraft.AppImage
```

Comment:
Rather than having 2 mostly identical install scripts, I would reccomend just picking the architecture with an if statement.

Reply:
thats my preference as well... and its own I implement it in my own scripts.
however botspot has the other preference